promise-polyfill-lite.html

```diff
@@ -7,10 +7,16 @@
 Code distributed by Google as part of the polymer project is also
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
-<link rel="import" href="../polymer/polymer.html">
 <script src='./Promise.js'></script>
 <script>
 if (!window.Promise) {
-  window.Promise = MakePromise(Polymer.Base.async);
+  var asap;
+  if (Polymer) {
```

Comment:
This is technically still a dependency on Polymer, no? Can we just replace it with what `webcomponents.js` did? https://github.com/webcomponents/webcomponentsjs/blob/v1/src/WebComponents/Promise.js#L130